AnnoDesigner/Localization/Localization.cs

```diff
@@ -87,6 +87,7 @@ public static void Init(ICommons commons)
                         { "ShowIcons" , "Show Icons" },
                         { "ShowTrueInfluenceRange" , "Show True Influence Range" },
                         { "ShowInfluences" , "Show Influences" },
+                        { "SelectAllBuildingsOfType" , "Select All Buildings of Type" },
```

Comment:
As noted the translation for the other languages is missing.
To prevent an error add the english text as a placeholder.